pineappl/src/grid.rs

```diff
@@ -124,17 +124,17 @@ bitflags! {
 #[derive(Clone, Deserialize, Serialize)]
 pub struct Grid {
     subgrids: Array3<SubgridEnum>,
-    channels: Vec<Channel>,
-    bin_limits: BinLimits,
+    bwfl: BinsWithFillLimits,
```

Comment:
Can't we call it just `bins` here please? We don't care about the implementation details inside grid

Reply:
It's not an implementation detail. Every bin has effectively two sets of limits:
1. The N-dimensional actual bin limits that also the user sees. These mustn't overlap, but they allow 'gaps' between them, also they must all have the same dimensions;
2. Fill limits: these are one-dimensional limits, the 'fill limits', that only `Grid::fill` uses. These are as many limits as there are bins plus one more and all values must be ascending.

Reply:
That being said I suppose we can find a better name. I'd like to avoid `Bins` though, because this struct has a method `fn bins()` and then the code would have a lot of `grid.bins().bins()` invocations, which is a bit odd.

Reply:
> That being said I suppose we can find a better name.

that was my main concern - `bins_config`?